campaignion_csv.module

```diff
@@ -112,6 +113,14 @@ function campaignion_csv_campaignion_csv_info() {
       'opt_in' => FALSE,
     ],
   ];
+  $export['target_message_count'] = [
+    'file_pattern' => [
+      'class' => YearlyFilePattern::class,
+      'path' => 'targets/%Y.csv',
+      'retention_period' => new \DateInterval('P24M'),
+      'refresh_interval' => new \DateInterval('PT23H30M'),
+    ],
+  ];
```

Comment:
Doesn’t this need exporter settings too? How does the export know to use the TargetStatsExporter?